doc/users.md

```diff
@@ -210,11 +210,12 @@ Move User's Folder
 ------------------
 
 To move all of a user's content to another user, call the
-[`moveFolderToUser(String destinationUserID)`][move-folder-to-user] method.
+[`transferContent(String destinationUserID)`][transfer-folder-to-new-user] method.
 
 ```java
-BoxUser user = new BoxUser(api, "0");
-BoxFolder.Info folderInfo = user.moveFolderToUser("1");
+String destinationUserID = "12345";
+BoxUser sourceUser = new BoxUser(api, "0");
```

Comment:
Maybe pull out a separate variable for `sourceUserID` to make this example more explicit